includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2167,11 +2167,8 @@ public function get_upe_enabled_payment_method_ids() {
 	 * @return string[]
 	 */
 	public function get_upe_available_payment_methods() {
-		return apply_filters(
```

Comment:
Before: we were allowing the dev tools to append additional payment methods when the UPE flag was disabled